bundles/org.openhab.binding.rfxcom/src/main/resources/ESH-INF/thing/rfxsensor.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
```

Comment:
Schema location needs update. Maybe rebase to get 0.8.0 SAT version and rerun to check xml files.

Reply:
The good thing is that I addressed this and the other thing is that I addressed some of the SAT findings. @Hilbrand would be great if you could do the first review. Thanks!